__tests__/api.test.js

```diff
@@ -24,4 +24,25 @@ describe('api/categories', ()=>{
             expect(result.body.length).toBe(4)
         })
     })
+})
+
+describe('api/reviews', ()=>{
+    test('responds with status code 200', ()=>{
+        return request(app).get('/api/reviews').expect(200)
+    })
+    test('responds with an array full of reviews with correct categories', ()=>{
+        return request(app).get('/api/reviews')
+        .then((result)=>{
+            result.body.forEach((review)=>{
```

Comment:
remember to test the array length is what you expect before testing with a forEach: avoids false positives